security-whitepaper/v2/cloud-native-security-whitepaper-it.md

```diff
@@ -0,0 +1,1335 @@
+# CLOUD NATIVE SECURITY WHITEPAPER | Version 2 - Italian Issue
+
+<!-- cspell:disable -->
+
+**Original Version in English**: 2.0 **Created**: 12 Jan 2022 **Status**: WIP | In Review | **Approved**
+
+**Last Reviewed**: 16 May 2022, **PDF Published**: 17 May 2022 **Release Version**: 2.0
+
+**Final PDF Approvers** [X] @TheFoxAtWork [X] @justincormack
+
+**Version 2 (May 2022)**
+
+- **Contributors**: Brandon Krieger, Cole Kennedy (TestifySec), Fatih Değirmenci (Ericsson Software Technology),
+
+Frederick Kautz , Joel Bork, Marina Moore (NYU), mateuszpruchniak@gmail.com, Pushkar Joglekar (VMware),
+
+raja.faisal@gmail.com, Savitha Raghunathan (Red Hat), Sayantani Saha (Independent)
+
+- **Reviewers**: Jon Zeolla, @nyrahul (Accuknox), Ragashree Shekar, Steven Hadfield, Kapil Bareja, Malini Bhandaru,
+
+Mikko Ylinen, Jonah Kowall (@jkowall), Kuang Dahu,Ariel Shuper, Eric Li (AlibabaCloud), Tanner Randolph (Applied
+
+Systems), Mark Dalton Gray (Microsoft), Ciara Carey, Brandon Lum (TAG Security Chair), Emily Fox (TOC Liaison) and
+
+Justin Cormack (TOC liaison).
```

Comment:
the markdown rendering of these lists is different from the original whitepaper due to the newlines.

Reply:
fixed thanks